src/main/kotlin/com/epam/brn/Application.kt

```diff
@@ -2,7 +2,9 @@ package com.epam.brn
 
 import org.springframework.boot.SpringApplication
 import org.springframework.boot.autoconfigure.SpringBootApplication
+import springfox.documentation.swagger2.annotations.EnableSwagger2
 
+@EnableSwagger2
```

Comment:
There is the next version of swagger: OpenApi 
https://medium.com/@tgtshanika/open-api-3-0-vs-swagger-2-0-94a80f121022

Did we decide to use swagger?

Reply:
does any other good alternatives to swagger?